app/docs/0.11.x/configuration.md

```diff
@@ -201,19 +206,45 @@ http {
 
 [Back to TOC](#table-of-contents)
 
-#### Embed Kong in OpenResty
+#### Starting Kong via Nginx
 
-If you are running your own OpenResty servers, you can also easily embed Kong
-by including the Kong Nginx sub-configuration using the `include` directive
+The command `kong start` launches Kong, internally initiating an Nginx
+process. If you are running your own OpenResty servers, you may want
+to flip this process over, starting Nginx yourself and having its
+configuration launch Kong. This may also desirable in containerized
+environments, to ensure that the Nginx master process is the foreground
+process in the container, instead of the `kong` CLI command.
+
+To launch Kong from within Nginx, you need an Nginx configuration file
+which includes the Kong Nginx sub-configuration using the `include` directive
 (similar to the examples of the previous section). However, you will need the
-final configuration and not the template. For this, use the `compile` command,
-which outputs a fully-compiled Nginx sub-configuration to `stdout`:
+final configuration and not the template. To prepare this configuration file,
+use the `kong prepare` command, which generates `nginx.conf` and
```

Comment:
Thanks for the feedback!

> This command also creates logs files, default self-generated SSL certificates

Only if they don't exist already, right? Re-running `kong prepare` does not seem to overwrite the files in `logs/` or `ssl/`.

> We can then say that once kong compile is run, all env variables and config files have been compiled and no further tweaking can occur. 

(You mean `kong prepare`, right?)

As I understand it, no further tweaking of `kong.conf` or `custom_nginx.template` takes effect after `kong prepare`, but the user may further tweak the generated `nginx.conf`, right? I alluded to this in later in the text. But I can make this clearer saying explicitly which files are and which ones aren't available for tweaking after `kong prepare`.